app/uk/gov/hmrc/tariffclassificationfrontend/connector/BindingTariffClassificationConnector.scala

```diff
@@ -34,7 +34,11 @@ import scala.concurrent.Future
 @Singleton
 class BindingTariffClassificationConnector @Inject()(appConfig: AppConfig, client: AuthenticatedHttpClient) {
 
-  private lazy val statuses: String = Seq(NEW, OPEN, REFERRED, SUSPENDED)
+  private lazy val statuses: String = Set(NEW, OPEN, REFERRED, SUSPENDED)
+    .map(_.toString)
+    .reduce( (a: String, b: String) => s"$a,$b" )
+
+  private lazy val liveStatuses: String = Set( OPEN, REFERRED, SUSPENDED)
```

Comment:
It doesn't seem correct that the frontend 'knows' what statuses constitute a 'live' one. Shouldn't knowledge like this live in the backend?

Reply:
The back end is more a data store in our architecture, it doesn't care about business logic, it just cares about storing and finding cases/events

Reply:
Also, what is considered "live" is different depending on what UI you are in, so it makes sense to keep the logic where the concern is